### DISCUSSION TITLE
AWS Waf logs

### DISCUSSION THREAD

**Discussion by usmanaa10** [wazuh/wazuh#13812] [General]
Hi Friends,
             Wazuh is successfully fetching logs from bucket with no error or warning, I run wodle  manually too and got the following result shown in the image
![image](https://user-images.githubusercontent.com/66419518/173047792-a00a02cb-3625-41c8-bf97-90d9dcbf86a5.png)
 

But unfortunately i am not seeing any result on dashboard

![image](https://user-images.githubusercontent.com/66419518/173048006-d9566133-45e0-4035-ac96-6c0825b0648f.png)

Kindly help me out in resolving this issue ASAP... my project is not moving forward

**Comment by Tostti** [comment]
Hello [usmanaa10](https://github.com/usmanaa10), apologies for the late response. 

Are you sending the logs directly to the S3 Bucket or via Kinesis? Although it is possible to send the logs directly to the bucket, for the Wazuh integration is mandatory to use Kinesis, as it alters the structure of the files adding extra fields that Wazuh uses. 

You can see more information about it in this documentation: 
https://documentation.wazuh.com/current/amazon/services/supported-services/waf.html?highlight=waf

Let me know if you have any questions or doubts.

Kind regards,
